src/TinyIoC/TinyIoC.cs

```diff
@@ -3716,23 +3716,26 @@ private ObjectFactoryBase GetParentObjectFactory(TypeRegistration registration)
 
             ObjectFactoryBase factory;
 
+            if (_Parent._RegisteredTypes.TryGetValue(registration, out factory))

```

Comment:
Any idea why this was moved out? It would be nice to have additional test coverage for any changes.

Reply:
@joaomsa ^^